### PR TITLE
Code Cleanup (Not 100% but better)

### PR DESCRIPTION
**Pull request by linKhehe**
Not tested.
General code cleanup and unused import purge.

### CODE REVIEW

**Comment by daggy1234** [comment]
Thanks, for fixing poor code

**Comment by wasi-master** [comment]
Yay

File: dagbot/bot.py [hunk 1]
```diff
@@ -28,62 +28,67 @@ async def get_prefix(bot, message):
...
-            "events",
-            "errors",
-            "developer",
-            "help"
+            "text", "fun", "newimag",
+            "reddit", "games", "util",
+            "whysomart", "animals", "memes",
+            "tags", "misc", "settings",
+            "ai", "events", "errors",
+            "developer", "help"
```
Comment (Fogapod) [inline dagbot/bot.py:66]: I don't think black will agree with these

Comment (linKhehe) [reply dagbot/bot.py:66]: Black isn't the end all be all of code style.

Comment (daggy1234) [reply dagbot/bot.py:66]: I agree, I personally plan on moving away from black. I will be doing some more formatting on my own!

Comment (Fogapod) [reply dagbot/bot.py:66]: As far as I know, repo author uses it as primary formatter

Comment (Fogapod) [reply dagbot/bot.py:66]: Alright

**Pull request merged**